receptor_sleep/worker.py

```diff
@@ -29,7 +36,12 @@ def __aiter__(self):
 def execute(message, config):
     loop = asyncio.get_event_loop()
     queue = ResponseQueue(loop=loop)
-    payload = json.loads(message.raw_payload)
+    try:
+        payload = json.loads(message.raw_payload)
+    except json.JSONDecodeError as err:
```

Comment:
Should this stop execution and reply with an error message? Or do what is doing, ignore the parsing issue and move on?

Reply:
The work.py module system should catch the exception and reply with the error details. Are you not seeing that? It's entirely doable to have this send its own error if we want. It's up to you.

Reply:
I am seeing I was just checking if it was a good idea to ignore the parsing issue and use the defaults.

I will revert this and log the exception and reraise the exception